.github/workflows/test_redshift_serverless.yml

```
@@ -0,0 +1,178 @@
+          echo "Creating schema: $SCHEMA_NAME"
+          PGPASSWORD='${{ secrets.REDSHIFT_SERVERLESS_TEST_PASSWORD }}' psql \
+            -h ${{ env.REDSHIFT_HOST }} \
+            -U ${{ secrets.REDSHIFT_SERVERLESS_TEST_USERNAME }} \
+            -d ${{ env.REDSHIFT_DATABASE }} \
+            -p 5439 \
+            -c "CREATE SCHEMA IF NOT EXISTS $SCHEMA_NAME;"
+          
+          echo "✅ Schema $SCHEMA_NAME created!"
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)